features/kafka/consumer/src/main/proto/kafka-consumer-events.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+option java_package = "org.opennms.features.kafka.consumer.events";
+option java_outer_classname = "EventsProto";
+
+// The values differ from the standard codes in OpenNMS
+// since proto3 enforces us to start at 0
+enum Severity {
+    INDETERMINATE = 0;
+    CLEARED = 1;
+    NORMAL = 2;
+    WARNING = 3;
+    MINOR = 4;
+    MAJOR = 5;
+    CRITICAL = 6;
+}
+
+message EventParameter {
+    string name = 1;
+    string value = 2;
+}
+
+message Event {
```

Comment:
@j-white @RangerRick   I brought most of the elements in `org.opennms.netmgt.xml.event.Event` to proto.
Let me know if this looks fine.